go.mod

```diff
@@ -231,12 +227,78 @@ require (
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/warnings.v0 v0.1.2 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
-	k8s.io/kube-openapi v0.0.0-20241212222426-2c72e554b1e7 // indirect
-	k8s.io/kubectl v0.32.2 // indirect
+	k8s.io/api v0.32.3
+	k8s.io/apiextensions-apiserver v0.32.3
+	k8s.io/apimachinery v0.32.3
+	k8s.io/apiserver v0.32.3
+	k8s.io/cli-runtime v0.32.3
+	k8s.io/client-go v0.32.3
+	k8s.io/component-base v0.32.3
```

Comment:
Nit: It feels off to me that these direct dependencies are now showing up in the `require` block with all of the indirect dependencies. Any idea why this is happening?

Reply:
I think layout is up to "golang.org/x/mod/modfile"

Reply:
It is odd that these are now with the `// indirect` dependencies.

Reply:
The placement of dependencies (direct vs. // indirect) in go.mod is managed by the `go mod tidy` command. Even if a module is specified in a replace directive, it might be listed as // indirect if no code in the main module directly imports packages from it. The replace still correctly forces Go to use the specified version for this transitive dependency.